ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -85,17 +87,19 @@ final class ToDoListViewModel: ObservableObject {
                                     toDoItems: todos.other))
                 }
                 state = .loaded(todoSections)
+                loading = false
             case .failure(let error):
                 state = .failed(error)
             }
         }
     }
     // Update the status of the task
     func updateStatus(todo: TaskModel) {
+        var task = todo
+        task.status.toggle()
```

Comment:
is there a reason why this code had to be changed?

Reply:
Oh because previously, I try to return the task with updated status to see whether the checkmark is toggled but it didn't work, I'll change it back